modules/claims_api/app/controllers/concerns/claims_api/poa_verification.rb

```diff
@@ -86,6 +86,33 @@ def poa_code_in_organization?(poa_code)
 
         representative.user_types.include?('veteran_service_officer')
       end
+
+      def verify_consent_limitations!
+        poa_code = BGS::PowerOfAttorneyVerifier.new(target_veteran).current_poa_code
+
+        form_code = if poa_code_in_organization?(poa_code)
+                      '21-22'
+                    else
+                      '21-22A'
+                    end
+        representatives = bgs_service.veteran_representative.read_all_veteran_representatives(
+          form_type_code: form_code, veteran_corp_ptcpnt_id: target_veteran.participant_id
+        )
+        # 2, possibly 3, scenarios to account for (are they naturally occurring or just a result of bad data in lower environments?):
+        # - what if it returns nil? (ie, no result for that form + id combo)
+        #     - e.g. Jesse Gray currently has this scenario
+        # - what if it returns content but none of it matches with the current POA code?
+        #     - e.g. Ralph E Lee and Greg Anderson have this situation
+        # - what if the form_code is a false positive?
+        #     - i.e., poa_code_in_organization? says it should be 21-22 but it's actually 21-22A
+        #     - Idk if this one actually happens or not
+        rep_poa_codes = representatives.pluck(:poa_code)
+
+        unless rep_poa_codes.include?(poa_code)
+          raise ::Common::Exceptions::Forbidden,
+                detail: 'Veteran has not granted access to records protected by Section 7332, Title 38, U.S.C.'
```

Comment:
- [ ] Update this to 404 (or something similar) with message (paraphrased) `Talk to the information office at your closest regional office`.

Basically, we want to:

1. Just show no claims without indicating that it's due to limited consent
2. Give actionable next steps (if there's a way they could theoretically access those claims)